src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -105,6 +105,10 @@ function getPaymentMethodTitle(
                 logoUrl: 'https://masterpass.com/dyn/img/acc/global/mp_mark_hor_blk.svg',
                 titleText: '',
             },
+            [PaymentMethodId.Opy]: {
+                logoUrl: cdnPath('/img/payment-providers/opy.png'),
```

Comment:
Nice job on implementing Openpay. Could I know how to put an icon file on CDN?

Reply:
I don't think it's a CDN as such, but the icon was added as an asset in the PR mentioned in the description above https://github.com/bigcommerce/bigcommerce/pull/42448